pkg/models/sidecar.go

```diff
@@ -39,6 +39,7 @@ type Sidecar struct {
 	RunRelayer        bool
 	// SubnetEVM based VM's only
 	SubnetEVMMainnetChainID uint
+	ValidatorManagement     ValidatorManagementType
```

Comment:
we may want to add a boolean to decide if using SoV (Subnet Only Validators) or not.
and this validator management refers to SoV, probably SoValidatorManagement .
Maybe:
- UseSoV
- SoValidatorManagement